app/views/admin/twitter_accounts/show.html.haml

```diff
@@ -1,8 +1,9 @@
 .admin-subnav
   .col-6
     .mr-4
-      = link_to "Edit", edit_admin_twitter_account_url(@twitter_account), class: "btn btn-lg btn-success"
-      = link_to "Check Credentials", check_credentials_admin_twitter_account_url(@twitter_account), class: "btn btn-lg btn-success"
+      = link_to "Edit", edit_admin_twitter_account_url(@twitter_account), class: "btn btn-success"
+      = link_to "Check Credentials", check_credentials_admin_twitter_account_url(@twitter_account), class: "btn btn-success"
+      = link_to "Log in to refresh credentials", "/auth/twitter", class: "btn btn-success"
```

Comment:
Rather than putting this on the show page, could you put it on the edit page? I think it might make sense to put in the account credentials box.

Maybe it makes sense to have in both places, but it definitely makes sense to have on the page next to the credentials.

Reply:
No prob -- updated in eb0518ee

<img width="806" alt="Screen Shot 2019-10-29 at 4 13 03 PM" src="https://user-images.githubusercontent.com/4433943/67805300-1dcc1400-fa67-11e9-9023-e8256363b2e0.png">
<img width="991" alt="Screen Shot 2019-10-29 at 4 13 12 PM" src="https://user-images.githubusercontent.com/4433943/67805301-1dcc1400-fa67-11e9-8d1a-7ac05674912c.png">
<img width="559" alt="Screen Shot 2019-10-29 at 4 13 19 PM" src="https://user-images.githubusercontent.com/4433943/67805302-1dcc1400-fa67-11e9-8d48-d58661b91946.png">